samples/excel/85-preview-apis/worksheet-page-layout.yaml

```diff
@@ -0,0 +1,198 @@
+                    ["G Farms", "Orange", "Organic", 200, 2200],
+                    ["H Farms", "Lime", "Conventional", 80, 1000],
+                    ["H Farms", "Lemon", "Conventional", 75, 1230],
+                    ["H Farms", "Orange", "Conventional", 25, 800],
+                    ["H Farms", "Orange", "Organic", 20, 500],
+                    ["H Farms", "Lemon", "Organic", 10, 770],
+                    ["H Farms", "Kiwi", "Conventional", 30, 300],
+                    ["I Farms", "Lime", "Organic", 50, 400],
+                    ["I Farms", "Apple", "Organic", 275, 220],
+                    ["I Farms", "Kiwi", "Organic", 200, 120],
+                    ["I Farms", "Apple", "Conventional", 100, 3000],
+                    ["J Farms", "Apple", "Organic", 80, 2800],
+                    ["J Farms", "Lime", "Conventional", 160, 2700],
+                    ["K Farms", "Orange", "Conventional", 180, 2000],
+                    ["K Farms", "Apple", "Conventional", 245, 2200],
+                    ["L Farms", "Kiwi", "Conventional", 200, 1500],
+                    ["L Farms", "Kiwi", "Organic", 100, 150],
+                    ["L Farms", "Lemon", "Conventional", 150, 270]
+                ]);
+
+                farmTable.getRange().format.autofitColumns();
+                printSheet.activate();
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to set page layouts for printing a worksheet.</p>
+        </section>
+        <section class="setup ms-font-m">
+          <h3>Setup</h3>
+          <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Create sample data</span>
+            </button>
+        </section>
+        <section class="setup ms-font-m">
+            <h3>Try it out</h3>
+            <p>Go to the <b>View > Page Layout</b> to see the layout changes as you press the buttons below.</p>
+            <button id="setPageBreaks" class="ms-Button">
+                <span class="ms-Button-label">Set page break</span>
+            </button><p/>
+            <button id="setPrintTitleRow" class="ms-Button">
+                <span class="ms-Button-label">Set title rows on each page</span>
+            </button><p/><p/>
+            <button id="center" class="ms-Button">
+                <span class="ms-Button-label">Center data</span>
+            </button>
+            <p>Go to the <b>File > Print</b> to see differences in printing before and after selecting the options below.</p>
```

Comment:
Consider rephrasing this instruction -> 

`<p>In Excel, choose <b>File > Print</b> to see differences in printing before and after you press the following buttons:</p>`